### ISSUE TITLE
Show the "rollback suggestion" button permanently

### ISSUE THREAD

**Issue by lukasrothenberger** [discopop-project/discopop-gui-vscode#43]
I think it might be beneficial to show the "rollback suggestion" button (i.e. the "X") in the suggestion list permanently, not only when hovering over the suggestion.
This would significantly simplify finding suggestions for reversion, in my opinion. 
It should not clutter the interface too much, since it is expected that only a small portion of the identified suggestions is implemented in the end.

EDIT: might be removed since applied state is now shown by the icons

**Comment by goerlibe** [comment]
I don't think it is possible to always show the icon. I would like to do this in quite a few places actually.

To make finding reversible suggestions easier we should implement a filter option for the tree view, so that only applied/unapplied/all suggestions are shown.

Also the icons for applied/unapplied should be more obviously different, maybe also use different colors.

What do you think about changing the label of the type in the tree to `type(x/y)`, where y is total amount of suggestions of that type and x is the amount of applied suggestions of that type.

Sidenote: maybe I could also rename the entire view from `suggestions` to `suggestions(x/y)`. I have to check if I can do that though. Possibly the view name is fixed in the package.json... It would be nice to show the total amount of suggestions somewhere to brag how great discopop is 😝

**Comment by goerlibe** [comment]
bragging is implemented

**Comment by lukasrothenberger** [comment]
`type(x/y),` seems like a good suggestion to me :) 

I'm not too sure about the usefulness of the "total amount" to be honest. 
Or rather: if we implement it for the suggestions, we should also show a similar view for the Hotspots (e.g. `(4,24,142)` )

**Comment by goerlibe** [comment]
superseded by #46 (filtering of suggestions) and #31 (custom icons)
